nsd-checkconf.c

```diff
@@ -361,6 +361,8 @@ config_print_zone(nsd_options_type* opt, const char* k, int s, const char *o,
 		/* bin */
 		SERV_GET_BIN(ip_transparent, o);
 		SERV_GET_BIN(ip_freebind, o);
+		SERV_GET_BIN(send_buffer_size, o);
```

Comment:
This part doesn't seem quite right yet as both `send_buffer_size` and `receive_buffer_size` are integer options as opposed to binary (*yes* or *no*) options. One fix coming up...